index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+
+    <script src="script.js"></script>
+</head>
+<body>
+    
+</body>
+</html>
```

Comment:
@GoutamShanbhag please add a new line at the end of file.
Please refer https://github.com/Surya-Training/Training/tree/master/git#newline-at-end-of-file to know more.